src/cli/index.ts

```diff
@@ -8,8 +8,12 @@ export const run = async () => {
   // global options
   program
     //.enablePositionalOptions(true)
-    .option("--config <config>", "Path to configuration file", "./config.hjson")
-    .option("-d, --dry-run", "Prints out rewards. Won't sumbit transactions.");
+    .option("-d, --dry-run", "Prints out rewards. Won't sumbit transactions.")
+    .option(
```

Comment:
Config was nice but could cause reports overwrite. Sticking with `--home` directories should provide nice way how to get clean and predictable environments. 

So one can operate folder structure as follows:
```sh
/bc
   |- mainnet
       |- config.hjson
       |- reports/...
       |- payout_wallet_private.key
    |- ithaca
       |- config.hjson
       |- reports/...
       |- payout_wallet_private.key
```
*So people won't be able to collide in any unexpected way.*

And btw @kalouo not sure whether `--home` is right terminology. We can use `-w,--work-dir` what do you think?

Reply:
Can I suggest we use `--network` instead and use `mainnet` as the default value?

Reply:
This also makes me think that the `network_configuration` object in the config can take different keys e.g.   

```json
"network_configuration": {
   "mainnet": {
    "rpc": "https://ithacanet.ecadinfra.com",
    "suppress_smartcontract_payments": true,
  } }
```

Reply:
But if you agree with above, let's please leave it for another PR otherwise will be a constant scope-creep.

Reply:
As we talked. This is not necessarily related to networks. So `-w,--work-dir` is the right flag for now.